doc/source/train/distributed-xgboost-lightgbm.rst

```diff
@@ -190,6 +190,47 @@ machines have 16 CPUs in addition to the 4 GPUs, each actor should have
     :end-before: __gpu_xgboost_end__
 
 
+.. _data-ingest-gbdt:
+
+How to preprocess data for training?
+------------------------------------
+
+The recommended way to preprocess data for training is to use Ray Data operations such as `map_batches`. 
```

Comment:
let's remove this sentence? same reasoning as previous comment

Reply:
addressed